Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.8-slim
+
+LABEL base_image="python:3.8-slim"
+LABEL about.home="https://github.com/Clinical-Genomics/NIPTool"
+LABEL about.tags="NIPT,statistics,Non Invasive Prenatal Test,python"
+
+WORKDIR /home/worker/app
+COPY . /home/worker/app
+
+# Install app requirements
+RUN pip install -r requirements.txt
+
+# Install app
+RUN pip install -e .
+
+# Create and switch to a new non-root user
+RUN useradd worker
+RUN chown worker:worker -R /home/worker
+USER worker
+
+ENTRYPOINT ["nipt"]
```

Comment:
I guess this will not be run as a CLI but a server, will that work in this way? Perhaps you could connect this to a nginx-container in docker-compose and try it?

Reply:
The command to run the server is `nipt run`, from the CLI, so I think it's safe to leave it like this for now. We can always change it later if we see that the app has problems to run from docker-compose. I'll write another PR with a docker-compose example!

Reply:
`nipt run` just starts the flask server. That should not be used in production, this is what I thought. Since we use containers it is fairly straight forward to connect this to a nginx container directly, don't you think?

Reply:
Sure, I'll try. I'll remove the entrypoint then!